apps/etterlatte-behandling/src/main/kotlin/tilgangsstyring/Tilgangsstyring.kt

```diff
@@ -138,12 +137,10 @@ suspend inline fun PipelineContext<*, ApplicationCall>.withFoedselsnummerInterna
     when (brukerTokenInfo) {
         is Saksbehandler -> {
             val harTilgang =
-                inTransaction {
-                    tilgangService.harTilgangTilPerson(
-                        foedselsnummer.value,
-                        Kontekst.get().appUserAsSaksbehandler().saksbehandlerMedRoller,
-                    )
-                }
+                tilgangService.harTilgangTilPerson(
```

Comment:
Samme her

Reply:
Denne er brukt som wrapper flere steder som kan være årsaken til at man får  ERROR: could not serialize access due to read/write dependencies among transactions